viewer/package.json

```diff
@@ -16,8 +16,9 @@
     "coverage": "jest --coverage",
     "lint": "tslint --project .",
     "serve": "npm install && npm link && cd examples && npm install && npm link @cognite/reveal && npm run serve",
-    "prepublishscript": "npm run release",
-    "publishscript": "node script/copyPackage.js && cd dist && npm publish"
+    "prepublishscript": "npm run release && npm run coverage",
+    "publishscript": "node script/copyPackage.js",
+    "postpublishscript": "cd dist && npm publish"
```

Comment:
we could probably have `"files": [ "dist/**/*" ],` specified to avoid using `cd dist` here.

Reply:
The reason is that if we do that the package.json publish will publish the dist folder inside the dist folder :) What we do want is to publish everything in the dist folder as a package.